### DISCUSSION TITLE
30x30 land protection implementation in GCAM

### DISCUSSION THREAD

**Discussion by jangoepel** [JGCRI/gcam-core#499] [Q&A] [answered]
Dear all,
my task is easily described but harder realized. Any input is welcome.
The task: 
restrict land use expansion to a level that conserves 30% of natural terrestrial land cover first for a single GCAM-land region and later globally
The problem:
1. As far as I have found out, the XML input files for scenario manipulation/policy settings requires me to set a limit to all natural land leafs (grassland, shrubland, unmanaged forest) for all individual regions in question in the sense that I apply a tax to all conversions from natural to anthropogenic use that would quantitatively subtract from the 30% conservation requirement. The problem is, is there even a way to set a relative limit (%) or do I have to extract the initial land supply absolute value, and calculate the 30% threshold from the original thousand square kilometer value and then input the 30% value in absolute terms (thsd sqkm)?
2. Are there any such studies (or scenarios) available that would make my work easier? I have searched for quite some time and haven't been lucky so far.
Again, any input/ideas is appreciated a lot.

**Comment by pkyle** [comment, marked as answer]
There was a core model proposal a few years ago that revamped how protected lands are handled, so I don't know for sure. Perhaps @kanishkan91  can weigh in.
I believe in constants.R, set
```
aglu.PROTECTION_DATA_SOURCE_DEFAULT <- TRUE
…
aglu.PROTECT_DEFAULT<- 0.3
```
Of course that approach would be a global value; assigning this at the regional level would require some revisions to the existing R code. And I don't expect the 30% protection fraction would really drive that much model behavior, as there would still be so much land available for conversion. The old default that we used, 90%, was selected in part because that was the scale of protection that was required to influence the model's behavior.

**Reply by jangoepel** [reply]
Thanks a lot, yes, as I see it doesn't have a meaningful impact on model behavior, at least on the global scale. I will have a closer look at regional levels.
